front/src/modules/keyboard-shortcut-menu/components/KeyboardShortcutMenuStyles.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from '@emotion/styled';
+
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+
+export const StyledDialog = styled.div`
```

Comment:
we should leverage our existing Dialog component here. You can take a look at the DialogManager